src/io/github/thebusybiscuit/cscorelib2/materials/MaterialTools.java

```diff
@@ -0,0 +1,40 @@
+package io.github.thebusybiscuit.cscorelib2.materials;
+
+import lombok.Getter;
+import org.bukkit.Material;
+import org.bukkit.Tag;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This utility class can be used to get {@link Material} Arrays that a tool is best at.
+ * Mainly separated by the class {@link Tag}
+ *
+ * @author TheBusyBiscuit
```

Comment:
I am the author? XD

You should update the description, as it is not so strongly related to the Tag Class.
And you should set your name as the author instead.

Reply:
oh right well that's a copy-paste lmao